index.php

```diff
@@ -36,7 +40,8 @@
 
                     <!-- perex -->
                     <p>
-                        <?php echo $clanek["perex"]; ?>
+                        <?php echo substr($clanek["perex"], 0, 133); ?>
+                        <?php if (strlen($clanek["perex"]) > 133) { echo "..."; } ?>
```

Comment:
Tohle se dela s odlisnymi cisly - abys nepsal tri tecky, kdyz ti zbyva 1 znak... Pripadne dneska uz se to resi tak, ze tam das delsi text, ale zkratis ho s tema 3 teckama pres CSS, ale to neni naplni tohodle kurzu _(ikdyz by nebylo od veci to zminit)_
```
<?php if (strlen($clanek["perex"]) > 125) {
  echo substr($clanek["perex"], 0, 133)."...";
else {
  echo $clanek["perex"]
}
?>
```

Reply:
No to já vim, ale na tom bychom si to nevyzkoušeli :)). Nebo jinej nápad, co smysluplného dělat se `substr`?

Reply:
Nemám problém s tím tvým kódem - jenom bych to tam zmínil a případně to můžeme dát jako součást cvičení na další hodinu

Reply:
Jak ať se to naučí v CSS :)?

Reply:
ne - jako at to overuju pres jiny cisla